src/content/containers/TimelineMarkers.css

```diff
@@ -0,0 +1,53 @@
+.timelineMarkersLabels {
+  width: 135px;
+  display: flex;
+  padding: 9px 0 9px 14px 0;
```

Comment:
Wow, it parsed it, threw away the last value, and applied the rule. Kind of surprised.